src/components/WizardFooter/WizardFooter.tsx

```diff
@@ -1,121 +1,89 @@
-import React, { Component } from 'react';
-import { ChevronLeft } from '@amsterdam/asc-assets';
+import React, { ComponentProps } from 'react';
 import {
-	CancelButtonStyle,
-	PreviousButtonStyle,
 	ButtonPanelStyle,
+	CancelButtonStyle,
 	LeftActionStyle,
+	PreviousButtonStyle,
 	RightActionStyle,
 	WizardButton,
 } from './WizardFooterStyles';
+import { ChevronLeft } from '@amsterdam/asc-assets';
```

Comment:
We might introduce an ESLint rule that ensures imports fro packages are imported ahead of relative path imports (considered best practice).

Reply:
Ah check, can change this manually for now. Or do you want me to create such an ESLint rule?

Reply:
Changed order of imports.